docs/SMCWG-charter.md

```diff
@@ -0,0 +1,334 @@
+# S/MIME Certificate Working Group (SMCWG) Charter
+
+## Introduction
+
+This introduction provides general information and context with an intent to
+assist the interpretation of this Charter.
+
+An S/MIME certificate contains a public key bound to an email address; and may
+also contain the identity of a natural person or legal entity that controls such
+email address. The key pair can then be used to sign, verify, encrypt, and
+decrypt email. An S/MIME certificate can be identified by the existence of an
+Extended Key Usage (EKU) Object Identifier (OID) of `1.3.6.1.5.5.7.3.4` for
+emailProtection.
+
+The objective of an S/MIME certificate is to provide cryptographic security
+services for electronic messaging applications, namely sender authentication,
+message integrity, and message privacy through encryption. For effective
+authentication and privacy, it is imperative that the CA validates the subject’s
+identity (if present) and its email address. The recipient of a digitally signed
+message can authenticate an email message to receive protection against email
+spoofing and can encrypt a response to the original sender by referring
+to the public key, email address, and distinguished name (if present) contained
+in an S/MIME certificate.
+
+The primary use case under consideration for the working group is a model
+whereby senders and recipients of email messages receive “reasonable assurance”
+that the other party to the communication identified in the certificate has
+control of the domain or email address being asserted. A variation of this
+primary use case is where an individual or organization digitally signs email to
+establish its authenticity and source of origin.
+
+Therefore, in order to provide reasonable assurance, it is crucial to establish
+a standard method to validate an email address and the subject’s identity (if
+present) prior to binding it them to the public key. “Reasonable
```

Comment:
I think the "it" needs to be dropped here.

Reply:
You can also use "suggestion" to propose an edit inline

```suggestion
present) prior to binding them to the public key. “Reasonable
```